docs: Update user guide (#58)
AY2324S1-CS2103T-T10-2/tp: docs/DraftUserGuide.md

<div style="page-break-after: always;"></div>

### 13. Clearing all task entries : `deleteAllTask`

{% include admonition.html type="danger" title="Potentially Dangerous Operation!" body="
AddressBook will discard all Task data and start with an empty data file at the next run.<br>" %}

Allows you to remove all entries from your task list.

Format:

```
deleteAllTask
```

Examples:

- `deleteAllTask`

Output:

![deleteAllTask success](images/output/deleteAllTask_success.png)

<div style="page-break-after: always;"></div>

### 14. Marking a task as completed: `markTask`

> Allows you to indicate that a specific task **has been finished**.

Reviewer: Perhaps you could replace `finished` with `completed` to standardise naming?

```suggestion
Allows you to indicate that a specific task **has been completed**.
```

Format:

```
markTask INDEX
```

- Marks the task at the specified `INDEX`.
- The index refers to the index number shown in the displayed task list.
- The index **_must be a positive integer_** 1, 2, 3, …

> {% include admonition.html type="info" title="Tasks start off as works in progress incomplete by default." %}
> 

Reviewer: Perhaps you could improve on the phrasing of this part?

```suggestion
{% include admonition.html type="info" title="

Tasks are marked as incomplete by default.

" %}
```

Examples:

- `listTask` followed by `markTask 2` marks the 2nd task in the task list as **completed**.
- `findTask Call` followed by `markTask 1` marks the 1st task in the results of the `findTask` command as **completed**.

Output:

![markTask_success](images/output/markTask_success.png)

Errors:

![markTask_error](images/error/markTask_error.png)

<div style="page-break-after: always;"></div>

### 14. Marking a task as incomplete: `unmarkTask`

> Allows you to indicate that a specific task **has not been finished**.

Reviewer: ```suggestion
Allows you to indicate that a specific task **has not been completed**.
```

Format:

```
unmarkTask INDEX
```

- Marks the task at the specified `INDEX`.
- The index refers to the index number shown in the displayed task list.
- The index **_must be a positive integer_** 1, 2, 3, …

> {% include admonition.html type="info" title="Tasks start off as works in progress incomplete by default." %}

Reviewer: ```suggestion
{% include admonition.html type="info" title="

Tasks are marked as incomplete by default.

" %}
```
